nixos/modules/programs/shadow.nix

```diff
@@ -60,7 +60,7 @@ in
   config = {
 
     environment.systemPackages =
-      lib.optional config.users.mutableUsers pkgs.shadow ++
+      lib.optional config.users.onlyDeclarative pkgs.shadow ++
```

Comment:
@rasendubi 
I'm sorry, but idk how to fix these.
Do I just put a ``!`` in front of ``config.users.onlyDeclarative``?